operations/commit_queue.go

```diff
@@ -173,27 +182,79 @@ func setModuleCommand() cli.Command {
 	}
 }
 
-func listCommitQueue(ctx context.Context, client client.Communicator, projectID string) error {
+func listCommitQueue(ctx context.Context, client client.Communicator, rc *legacyClient, projectID string, uiServerHost string) error {
 	cq, err := client.GetCommitQueue(ctx, projectID)
 	if err != nil {
 		return err
 	}
+	projectRef, err := rc.GetProjectRef(projectID) // rc versus ac?
+	if err != nil {
+		return errors.Wrapf(err, "can't find project for queue id '%s'", projectID)
+	}
+	grip.Infof("Project: %s\n", projectID)
+	grip.Infof("Type of queue: %s\n", projectRef.CommitQueue.PatchType)
+
+	if projectRef.CommitQueue.PatchType == commitqueue.PRPatchType {
+		grip.Infof("Owner: %s\n", projectRef.Owner)
+		grip.Infof("Repo: %s\n", projectRef.Repo)
+	}
 
-	grip.Infof("Project: %s\n", restModel.FromAPIString(cq.ProjectID))
 	grip.Infof("Queue Length: %d\n", len(cq.Queue))
 	for i, item := range cq.Queue {
-		grip.Infof("\t%d: %s\n", i+1, restModel.FromAPIString(item.Issue))
-		if len(item.Modules) > 0 {
-			grip.Info("\tModules:\n")
-			for j, module := range item.Modules {
-				grip.Infof("\t\t%d: %s (%s)\n", j+1, restModel.FromAPIString(module.Module), restModel.FromAPIString(module.Issue))
-			}
+		grip.Infof("%d:", i+1)
+		if projectRef.CommitQueue.PatchType == commitqueue.PRPatchType {
+			listPRCommitQueueItem(ctx, item, projectRef, uiServerHost)
+		}
+		if projectRef.CommitQueue.PatchType == commitqueue.CLIPatchType {
+			listCLICommitQueueItem(ctx, item, rc, uiServerHost)
 		}
+		listModules(item)
 	}
 
 	return nil
 }
 
+func listPRCommitQueueItem(ctx context.Context, item restModel.APICommitQueueItem, projectRef *model.ProjectRef, uiServerHost string) {
+	prDisplay := `
+        PR # : %s
+         URL : %s
+`
+	prDisplayVersion := "       Build : %s/version/%s"
+
+	issue := restModel.FromAPIString(item.Issue)
+	url := fmt.Sprintf("https://github.com/%s/%s/pull/%s", projectRef.Owner, projectRef.Repo, issue)
+	grip.Infof(prDisplay, issue, url)
+
+	if restModel.FromAPIString(item.Version) != "" {
+		grip.Infof(prDisplayVersion, uiServerHost, restModel.FromAPIString(item.Version))
+	}
+	grip.Info("\n")
+}
+
+func listCLICommitQueueItem(ctx context.Context, item restModel.APICommitQueueItem, rc *legacyClient, uiServerHost string) {
+	issue := restModel.FromAPIString(item.Issue)
+	p, err := rc.GetPatch(issue)
+	if err != nil {
+		grip.Error(message.WrapError(err, "\terror getting patch"))
+		return
+	}
+	disp, err := getPatchDisplay(p, false, uiServerHost)
+	if err != nil {
+		grip.Error(message.WrapError(err, "\terror getting patch display"))
+		return
+	}
+	grip.Info(disp)
+}
+
+func listModules(item restModel.APICommitQueueItem) {
+	if len(item.Modules) > 0 {
+		grip.Info("\tModules:\n")
+		for j, module := range item.Modules {
+			grip.Infof("\t\t%d: %s (%s)\n", j+1, restModel.FromAPIString(module.Module), restModel.FromAPIString(module.Issue))
```

Comment:
does this line up with the space indenting (and are tabs a consistent number of spaces)?

Reply:
I've updated to make them more consistent. I believe tabs are consistently 8 spaces (which lines the numbers up to the colon).

Reply:
I think the size of tabs is configurable:
```
$ echo -e "\thello"
        hello
$ tabs 4
$ echo -e "\thello"
    hello
```
Though I agree it's unusual for someone to adjust it.

Reply:
Whoa solid example. I'm fine with relying on tabs since the modules list will all still be indented consistently, unless someone has an objection.